content/organization/intro/partnerships/index.en.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Partnerships"
+menuTitle: "Partnerships"
+weight: 99
+---
+
+As stated in our [mission](/about/mission/), R-Ladies seeks to remain independent and non commercial.
+
+No official/formal connection with the R-Ladies organization in general can be advertised without agreement of the global team/leadership (send an email to `leadership@rladies.org`).
+
+Before that, how do you as an organizer or organizers team decide that a partnership is worth pursuing?
+It is important to assess whether the organization is looking for diversity points.
+You might ask others for their input, especially others who have collaborated with the same organization and who do not have conflicts of interest (so, not a staff member of that organization!).
+You can also look or ask whether the organization includes and has been supporting groups that have been systematically excluded from the tech scene.
+
+## Special case of conferences
+
+If you are invited to give a talk,
+* If you are invited to talk not about R-Ladies, check that they invite you because of what you do not only because you are an R-Ladies. 
```

Comment:
```suggestion
* If you are invited to talk about a topic other than R-Ladies, make sure that you are invited because of your expertise and not just your relationship to R-Ladies.
```